imports/client/components/HuntListPage.tsx

```diff
@@ -135,39 +136,28 @@ const DiscordChannelSelector = (params: DiscordSelectorParams) => {
   );
 };
 
-const DiscordRoleSelector = (params: DiscordSelectorParams) => {
+const DiscordRoleSelector = (params: DiscordSelectorParams & { guildId?: string }) => {
   const { ready, options } = useTracker(() => {
     const handle = Meteor.subscribe('discord.cache', { type: 'role' });
-    const discordRoles: SavedDiscordObjectType[] = DiscordCache.find(
-      { type: 'role' },
+    const discordRoles: SavedDiscordObjectType[] = DiscordCache.find({
+      type: 'role',
+      'object.guild': params.guildId,
```

Comment:
Same question here -- do you want this to enumerate all roles of all guilds if guildId is unset, or do you want to require that guildId be set first?